cfgov/cfgov/urls.py

```diff
@@ -41,6 +38,16 @@
 from v1.views.documents import DocumentServeView
 
 
+try:
+    from flags.path import flagged_re_path
+    from django.urls import include, re_path, static
```

Comment:
`static` is still in `django.conf.urls`.

Reply:
Good catch. Fixed static import path.